lib/helpers.ts

```diff
@@ -60,6 +60,8 @@ export interface BaseSerializableEntity {
     WPAtlas: WPAtlas;
     primaryParents?: HTANDataFileID[];
     synapseId?: string;
+    isRawSequencing?: boolean;
+    downloadSource?: 'dbGaP' | 'IDC' | 'Synapse' | 'Coming Soon';
```

Comment:
might be worth adding a `DownloadSource` enum. I see these values hard-coded in multiple places.

Reply:
Good point! Added an enum - lmk if that looks ok